test/distributed/test_c10d_error_logger.py

```diff
@@ -107,35 +47,31 @@ def failed_broadcast_not_raise_exception(self):
         except Exception as exception:
             pass
 
-    @with_comms
     def test_exception_handler_with_dist(self) -> None:
+        print(dist.get_rank())
```

Comment:
remove print?

Reply:
Ye. Thanks! Will clean up the prints after figuring out the hanging issue.